compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -130,6 +130,7 @@ pub mod visit;
 pub mod vtable;
 pub mod walk;
 
+mod _sty;
```

Comment:
```suggestion
mod sty_;
```
Items starting with underscore are typically used to silence `unused` lints, it's not an idiomatic naming.